proto/tendermint/types/canonical.proto

```diff
@@ -35,5 +35,4 @@ message CanonicalVote {
   CanonicalBlockID          block_id  = 4 [(gogoproto.customname) = "BlockID"];
   google.protobuf.Timestamp timestamp = 5 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
   string                    chain_id  = 6 [(gogoproto.customname) = "ChainID"];
-  CanonicalPartSetHeader part_set_header = 7;
```

Comment:
removal of the `PartSetHeader` from the `CanonicalVote`

Reply:
So the `CanonicalPartSetHeader` is still needed bc/ we have to sign over it in the proposal at least, right?

Reply:
@liamsi, yes. I double-checked and `CanonicalPartSetHeader` is only used in the canonical proposal.